cmd/queue/main.go

```diff
@@ -114,6 +115,10 @@ func initEnv() {
 func reportStats(statChan chan *autoscaler.Stat) {
 	for {
 		s := <-statChan
+		if !userContainerDeployed {
+			logger.Warnf("The user container is not ready yet.")
```

Comment:
Please include some context into the log, i.e: "Not reporting metrics, user-container is not ready yet."